addons/website/views/website_views.xml

```diff
@@ -297,6 +297,16 @@
             </field>
         </record>
 
+        <record id="reset_view_arch_wizard_view" model="ir.ui.view">
+            <field name="model">reset.view.arch.wizard</field>
+            <field name="inherit_id" ref="base.reset_view_arch_wizard_view"/>
+            <field name="arch" type="xml">
+                <field name="compare_view_id" position="attributes">
+                    <attribute name="context">{'display_website': True}</attribute>
```

Comment:
so perfect, but will be so more simple to let us add website stuff in base. :/

Reply:
Yeah don't tell me.. :cry: